concepticondata/concepticon.tsv

```diff
@@ -3034,3 +3034,31 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3038	SAME-SEX YOUNGER SIBLING	Kinship	A person of the same sex who is born after another person who has the same parents.	Person/Thing	
 3039	DIFFERENT-SEX OLDER SIBLING	Kinship	A person of the different sex who is born before another person who has the same parents.	Person/Thing	
 3040	DIFFERENT-SEX YOUNGER SIBLING	Kinship	A person of the different sex who is born after another person who has the same parents.	Person/Thing	
+3041	PATERNAL AUNT (FATHER'S OLDER SISTER)	Kinship	The older sister of one's father.	Person/Thing	
+3042	MATERNAL AUNT (MOTHER'S OLDER SISTER)	Kinship	The older sister of one's mother.	Person/Thing	
+3043	MATERNAL AUNT (MOTHER'S YOUNGER SISTER)	Kinship	The younger sister of one's mother.	Person/Thing	
+3044	BOVINE	Animals	Member of the Bovinae subfamily, mostly pointing to bulls, cows and oxes.	Person/Thing	
+3045	COPPER (COLOR)	Sense perception	Having a red to yellow metallic color that resembles the metal copper.	Property	
+3046	FODDER	Food and drink	Feed for livestock, especially chopped hay or straw.	Person/Thing	
+3047	HOT SEASON	Time	Period of the year associated with hot or warm weather.	Person/Thing	
+3048	MOUTH HARP	Modern world	Lamellophone instrument consisting of a tongue or reed attached to a frame, placed in the performer's mouth and plucked; Jew's harp, jaw harp.	Person/Thing	
+3049	OIL (FROM ANIMALS)	Food and drink	Liquid hydrophobic substance from animals which is used for cooking.	Person/Thing	
```

Comment:
Oil from animals should be FAT (FROM ANIMALS), 1282, because you usually have this in a non-liquid form, and in the Chinese language for glossing, they don't distinguish between OIL and FAT, so please link to 1282

Reply:
Oh, that was actually a left-over from the "same concept with different glosses", it is not used in Sun's list anymore.